README.md

```diff
@@ -0,0 +1,43 @@
+
+# Minishell
+Write a simple version of a shell.
+<br/>
+
+## TO DO
+* Figure out how we need to parse the commands => do all commands need to be executable (even non-built-in?), so do we need to figure out how to parse the options (and see if they are executable)?
+* Display a prompt (loop GNL). Print to stderr? (>msh)
+* Have a working history (cf [add_history function](https://linux.die.net/man/3/history))
+* Get the PATH variable (cf [pipex](https://github.com/Knulpinette/Cursus42/blob/main/02-pipex/srcs/utils.c))
+* No need to interpret \ or ; (unless required by the subject). Figure out when it is required by the subject lol

```

Comment:
hein? \ and ; are never required by the subject aha

Reply:
Well, they say they aren't unless they are so I still put it in case they are somehow somewhere required (that would be very 42 of them).